pkg/apis/kubeone/v1beta2/types.go

```diff
@@ -223,6 +223,10 @@ type HostConfig struct {
 	// Default value is "".
 	SSHPrivateKeyFile string `json:"sshPrivateKeyFile,omitempty"`
 
+	// SSHCertFile is path to the file with the certificate of the private key.
+	// Default value is "".
+	SSHCertFile string `json:"sshCertFile,omitempty"`
+
```

Comment:
Please run:

```shell
./hack/update-codegen.sh
./hack/update-apidocs.sh
```

and commit the results

Reply:
I couldn't run the `update-codegen.sh` because I got the following error

```
go generate ./pkg/...
go generate ./test/...
# k8c.io/kubeone/pkg/apis/kubeone
../../pkg/apis/kubeone/register.go:49:3: cannot use &KubeOneCluster{} (value of type *KubeOneCluster) as "k8s.io/apimachinery/pkg/runtime".Object value in argument to scheme.AddKnownTypes: *KubeOneCluster does not implement "k8s.io/apimachinery/pkg/runtime".Object (missing method DeepCopyObject)
test/e2e/tests_definitions.go:17: running "go": exit status 1
make: *** [Makefile:136: gogenerate] Error 1
```

Reply:
Sorry, I meant

```shell
make update-codegen
```

Reply:
@kron4eg Unfortunately I always get the same error. I don't know what I'm doing wrong here.

```
go version go1.21.4 darwin/arm64
go mod vendor
./hack/update-codegen.sh
examples/terraform/aws-dualstack/README.md
examples/terraform/aws/README.md
examples/terraform/azure/README.md
examples/terraform/digitalocean/README.md
examples/terraform/equinixmetal/README.md
examples/terraform/gce/README.md
examples/terraform/hetzner/README.md
examples/terraform/nutanix/README.md
examples/terraform/openstack/README.md
examples/terraform/vmware-cloud-director/README.md
examples/terraform/vsphere/README.md
examples/terraform/vsphere_centos/README.md
examples/terraform/vsphere_flatcar/README.md
WARNING: generate-internal-groups.sh is deprecated.
WARNING: Please use k8s.io/code-generator/kube_codegen.sh instead.

Generating deepcopy funcs
W0204 22:36:16.300253   28296 parse.go:863] Making unsupported type entry "T" for: &types.TypeParam{check:(*types.Checker)(nil), id:0x1, obj:(*types.TypeName)(0x1400bdc6be0), index:0, bound:(*types.Named)(0x1400001e1c0)}
Generating defaulters
W0204 22:36:18.169497   28399 parse.go:863] Making unsupported type entry "T" for: &types.TypeParam{check:(*types.Checker)(nil), id:0x1, obj:(*types.TypeName)(0x1400ae6cfa0), index:0, bound:(*types.Named)(0x14000124150)}
Generating conversions
W0204 22:36:20.028303   28505 parse.go:863] Making unsupported type entry "T" for: &types.TypeParam{check:(*types.Checker)(nil), id:0x1, obj:(*types.TypeName)(0x1400bddf040), index:0, bound:(*types.Named)(0x1400001e1c0)}
go generate ./pkg/...
go generate ./test/...
# k8c.io/kubeone/pkg/apis/kubeone
../../pkg/apis/kubeone/register.go:49:3: cannot use &KubeOneCluster{} (value of type *KubeOneCluster) as "k8s.io/apimachinery/pkg/runtime".Object value in argument to scheme.AddKnownTypes: *KubeOneCluster does not implement "k8s.io/apimachinery/pkg/runtime".Object (missing method DeepCopyObject)
test/e2e/tests_definitions.go:17: running "go": exit status 1
make[1]: *** [gogenerate] Error 1
make: *** [update-codegen] Error 2
```

Reply:
@kron4eg now i got it.